packages/xrpl/src/models/transactions/oracleSet.ts

```diff
@@ -0,0 +1,176 @@
+import { ValidationError } from '../../errors'
+
+import {
+  BaseTransaction,
+  isNumber,
+  isString,
+  validateBaseTransaction,
+  validateOptionalField,
+  validateRequiredField,
+} from './common'
+
+const PRICE_DATA_SERIES_MAX_LENGTH = 10
+
+/**
+ * A PriceData object represents the price information for a token pair.
+ *
+ */
+export interface PriceData {
+  PriceData: {
```

Comment:
Is there a docstring for `PriceData`?

Reply:
I don't see one for the inner `PriceData` and I thought it would be redundant to add it.